oak_containers_orchestrator/src/container_runtime.rs

```diff
@@ -101,84 +80,47 @@ pub async fn run(
         base
     };
 
-    // mount the utility dir (includes the ipc socket) into the container
-    let container_dev_path = {
-        let mut base = container_rootfs_path.clone();
-        base.push(crate::UTIL_DIR);
-        base
-    };
-    rmount_dir(
-        std::path::Path::new("/").join(crate::UTIL_DIR),
-        container_dev_path,
-    )
-    .await?;
-
-    // mount host /dev into the container
-    let container_dev_path = {
-        let mut base = container_rootfs_path.clone();
-        base.push("dev");
-        base
-    };
-    rmount_dir(std::path::PathBuf::from("/dev"), container_dev_path).await?;
-
-    // mount host /sys into the container
-    let container_sys_path = {
-        let mut base = container_rootfs_path.clone();
-        base.push("sys");
-        base
-    };
-    rmount_dir(std::path::PathBuf::from("/sys"), container_sys_path).await?;
-
-    // mount host /proc into the container
-    let container_proc_path = {
-        let mut base = container_rootfs_path.clone();
-        base.push("proc");
-        base
-    };
-    rmount_dir(std::path::PathBuf::from("/proc"), container_proc_path).await?;
-
     log::debug!(
         "Startup command: {:?}",
         oci_filesystem_bundle_config.process.args
     );
+
+    // systemd-nspawn claims to support OCI bundles, but unfortunately it doesn't support bundles
+    // in the 1.0.2-dev format. See https://github.com/systemd/systemd/issues/22148
```

Comment:
We should probably make it explicit in the comment that because of this, we don't actually pass the OCI bundle to nspawn, but instead parse its config file ourselves.